README.md

```diff
@@ -41,13 +43,32 @@ In your body you may include the component like:
 
 ```twig
 {# The Twig version of '@livewire' #}
-{% livewire counter %}
+{% livewire 'counter' %}
 
 {# If you wish to pass along variables to your component #}
-{% livewire counter with {'count': 3} %}
+{% livewire 'counter' with {'count': 3} %}
 
 {# To include a nested component (or dashes), you need to use '' #}
 {% livewire 'nested.component' %}
+
+{# To use key tracking, you need to use key(<expression>) #}
+{% livewire 'counter' key('counterkey') %}
+
+{# The Twig version of '@persist' #}
+{% persist 'name' %}
```

Comment:
I'm leaning against that all `livewire` twig components should be prefixed with `livewire`. Perhaps like:

`{% livewire.this %}`, if that's possible with Twig syntax.

What do you think about that?

Reply:
> I'm leaning against that all `livewire` twig components should be prefixed with `livewire`. Perhaps like:
> 
> `{% livewire.this %}`, if that's possible with Twig syntax.
> 
> What do you think about that?
It looks nice, yes. However, that dot is interpreted as punctuation, not as part of de name of the component. The livewire component would contain the logic of all the dot-separated components in one tokenparser. Also some trickery required to implement `entangle` ... `endentangle`

Reply:
Something like `livewire.[]` seems a bit difficult to implement.
Perhaps using a function (`livewire('this')` ? `lw('this')`) could be simpler to implement, and stay intuitive ?